packages/admin/admin/src/dataGrid/muiGridSortToGql.tsx

```diff
@@ -0,0 +1,12 @@
+import { GridSortModel } from "@mui/x-data-grid";
+
+export function muiGridSortToGql(sortModel?: GridSortModel) {
+    if (!sortModel) return undefined;
+    return sortModel.map((i) => {
+        return {
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            field: i.field as any, //as any to be compatible with enum
```

Comment:
You could cast `as string` to avoid the lint disable

Reply:
no: `Type 'string' is not assignable to type 'GQLProductSortField`